mapgen.lua

```diff
@@ -886,7 +886,7 @@ minetest.register_abm({
 	function(pos, node)
 		for _,obj in ipairs(minetest.get_objects_inside_radius(pos, 5)) do
 			if obj:is_player() then
-				explosion(pos, 3, 0, 1)
+				tnt.boom(pos,{radious = 3,dmage_radious = 0})
```

Comment:
Should be `radius` and `damage_radius`, surely?

Reply:
sound ilogic but it damages me! XD also this is the only active workiing one.. i try to track and sync the changes with tenplush, if you wants..  we can sync and make the tenplush one the oficial.. the nssb is working with the fork from him!

what we can do?

Reply:
I mean, you mis-spelled the variables

[Look at the PR you linked](https://github.com/minetest/minetest_game/pull/1903/files)

The variables are called `radius` and `damage_radius` . Your PR is listing `radious` and `dmage_radious`

Reply:
errrr UPSSSSS @taikedz :face_exhaling:  .. ok ok got it! (in my fork does not fail cos i used the "main" branch  :face_with_spiral_eyes:  let me solve it!